tests/core/fs/test_fs_sync.py

```diff
@@ -582,7 +581,8 @@
 
     # First create data locally
     with freeze_time("2000-01-02"):
-        obj_id = await alice_fs.file_create("/w/foo.txt")
+        fd = await alice_fs.file_create("/w/foo.txt")
```

Comment:
Useful to have `file_create` returning the entry id

Given returning file descriptor is also useful, I would prefer to be able to do:
```python
file_id = fs.file_create("/foo.txt")
file_id, fd = fs.file_create("/bar.txtb, open=True)
```

Reply:
Is there a use case I'm not aware of? Because it wasn't used anywhere in the code base. By the way `folder_create` no longer returns the entry id either.

As far as I understand the concept of entry id only leaks out of the FS because we want to send notifications to other devices without using plain path information right?

Also I can add the `open` flag to `file_create`, but I don't think it's going to be used in the code base, except maybe for some tests.

Reply:
Done! (see e55ed7f and 1c3d68c)